backstop/tests/toast-notification.html

```diff
@@ -265,5 +265,44 @@ <h1>Toast notification</h1>
       </div>
 
     </div>
+    <div class="iui-toast-wrapper iui-placement-bottom-end">
+
+      <div
+        class="iui-toast iui-positive"
+        id="demo-positive"
```

Comment:
Looks like this id is duplicate so it's producing lots of extra images

Reply:
I would just remove the id if you're not using it.

Also there are still duplicate images because this selector appears twice in the DOM:
https://github.com/iTwin/iTwinUI/blob/e448f3fd9bf5bbc017fc2a193010ad32b036055e/backstop/scenarios/toast-notification.js#L6

You could either update the scenario or make sure there is only one wrapper in the html.

Reply:
Removed additional toaster-wrapper component, added JS to switch between placements. Added tests for those placements

Reply:
I think there are still a couple left over images. Can you verify? Run `yarn test` followed by `yarn clean:images`